packages/core/state/selection/actions.ts

```diff
@@ -157,15 +157,19 @@ export const DESELECT_DISPLAY_ANNOTATION = makeConstant(
 );
 
 export interface DeselectDisplayAnnotationAction {
-    payload: Annotation | Annotation[];
+    payload: {
```

Comment:
? why

Reply:
I thought this made it a little easer to read mirroring the SelectDisplayAnnotationAction so that both have action.payload.annotation as the annotation rather than action.payload for one and action.payload.annotation for another